pkg/bootstrap/envoy_bootstrap.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2018 Istio Authors
```

Comment:
does this need a dedicated package? can you update owners file to assign ownership to pkg/bootstrap?

Reply:
It will be shared with node-agent, doesn't need additional owners, same rule as all shared packages. I think bikeshedding the directory name doesn't help - I did think about this and I believe this is the best location. If we add nginx we can add another file to generate nginx bootstrap here.

Reply:
Ok. I don't have a strong opinion about package, just not loosing association with envoy.